src/modules/sub-items/sub.items.module.js

```diff
@@ -293,72 +447,95 @@ export default class SubItemsModule extends Component {
         this.setState(() => ({ activeView }));
     }
 
-    toggleItemSelect(location, isSelected) {
-        const { selectedLocationsIds } = this.state;
-        const updatedSelectedItemsIds = new Set(selectedLocationsIds);
+    /**
+     * Returns data which we want to cache for selected item
+     * Page data might be discarded but selection of the item should stay,
+     * thus we need item data to perform operations such as bulk delete or bulk move
+     *
+     * @param {Object} item
+     * @param {Object} item.content
+     * @param {Object} item.location
+     *
+     * @returns {Object} dataToCache
+     */
+    getItemDataToCache(item) {
```

Comment:
I don't get the idea of caching here. Can you elaborate more on this topic? What is the flow?

Reply:
You cache item data during a selection of a page item so that you have item data even if page data were discarded.